src/common/entity/domain_icon.ts

```diff
@@ -63,6 +65,9 @@ export const domainIcon = (
     case "media_player":
       return compareState === "playing" ? mdiCastConnected : mdiCast;
 
+    case "switch":
+      return compareState === "on" ? mdiToggleSwitch : mdiToggleSwitchOff;
```

Comment:
I don't think we should use this for all switches, for those with device_class `toggle` this makes sense.
For `outlet`? maybe `mdiPowerPlug` and `mdiPowerPlugOff`
For switches with assumed state, I think the current is the best variant.

Reply:
tbh, Ive never understood why the flash was used at all, even using assumed state, having 2 identical flashes is very unclear in its meaning.
could you please point me to the switch device_class, I've just asked about that in Discord #frontend, as I applied it and no change in icon occurs.

agree powerful would be cool to add for those
can we add:
```
if (stateObj?.attributes.device_class === "outlet") {
         return compareState === "on" ? mdiPowerPlug : mdiPowerPlugOff;
       }
```
like that?

Reply:
it was `switch` I meant, updated.